packages/ddg2dnr/README.md

```diff
@@ -28,7 +28,7 @@ Generate the TDS ruleset:
 
 ```bash
 npm run tds ../tds-input.json ../supported-surrogates-input.json ../tds-ruleset-output.json \
-        [../match-details-by-rule-id-output.json]
+        [../allowing-rules-by-ctl-action-output.json] [../match-details-by-rule-id-output.json]
```

Comment:
Are these both individually optional, or you can either specify none or both? If the latter, I think it makes more sense to use a single set of square brackets over both terms. 

If they are individually optional, it's not actually possible to provide `match-details-by-rule-id-output` without `allowing-rules-by-ctl-action-output`.

Reply:
That's right, you have to specify the first if you want to specify the second. I think that's fine honestly.